src/electron/process_manager.ts

```diff
@@ -76,8 +76,9 @@ const UDP_FORWARDING_TEST_RETRY_INTERVAL_MS = 1000;
 //
 // Fulfills with a copy of `serverConfig` that includes the resolved hostname.
 export function startVpn(
-  serverConfig: cordova.plugins.outline.ServerConfig, onDisconnected: () => void,
-  isAutoConnect = false): Promise<cordova.plugins.outline.ServerConfig> {
+    serverConfig: cordova.plugins.outline.ServerConfig, onDisconnected: () => void,
```

Comment:
It seems like there's some redundancy in the two handlers. Did you think about killing `onDisconnected` in favour of listening for reconnection and disconnection events in `onConnectionStatusChange`?

Reply:
Good suggestion, done in 3ecaa6b.